blog/_posts/2021-06-17-uncertainty.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: review
+title: "Uncertainty Estimation Review"
+tags: machine-learning essentials blog
+author: Thierry Judge
+---
+
+# Introduction
+
+This post is a review of uncertainty estimation methods with an emphasis on classification and segmentation. 
+
+# Evaluating Uncertainty
+
+## Model calibration
+
+A neural network trained for classification is said to be calibrated if its confidence is equal to the probability of 
+it being correct. For example, given 100 predictions with an average confidence of 0.7, 70 predictions should be correct. 
+This can be expressed :
+
+$$
+\mathbb{P} ( \hat{Y} = Y | \hat{P} = P ) = p  \text{   } \forall p \in [0, 1]
```

Comment:
Was the `\text{   }` statement intended to contain something? If you want to introduce whitespaces in math mode, there are some specific commands to do so `\;`, `\quad`: https://www.overleaf.com/learn/latex/Spacing_in_math_mode

Reply:
Done in eacdc9f.